api/notd/store/retriever.py

```diff
@@ -177,3 +180,15 @@ async def list_collection_tokens_by_owner(self, address: str, ownerAddress: str,
         for token in soldTokens:
             boughtTokens.remove(token)
         return list(boughtTokens)
+
+    async def get_collection_activity(self, address: str, connection: Optional[DatabaseConnection] = None) -> List[CollectionActivity]:
+        startDate =  date_util.start_of_day(dt=datetime.datetime.now())
+        endDate =  date_util.start_of_day(dt=date_util.datetime_from_datetime(dt=startDate, days=-90))
+        query = select([TokenTransfersTable.c.registryAddress, sqlalchemyfunc.sum(TokenTransfersTable.c.amount), sqlalchemyfunc.sum(TokenTransfersTable.c.value), sqlalchemyfunc.date(BlocksTable.c.blockDate)]).join(BlocksTable, BlocksTable.c.blockNumber == TokenTransfersTable.c.blockNumber)
+        query = query.where(TokenTransfersTable.c.registryAddress == address)
+        query = query.where(BlocksTable.c.blockDate < startDate)
+        query = query.where(BlocksTable.c.blockDate >= endDate)
+        query = query.group_by(sqlalchemyfunc.date(BlocksTable.c.blockDate),TokenTransfersTable.c.registryAddress)
+        result = await self.database.execute(query=query, connection=connection)
+        collectionGraph = [collection_graph_from_row(row) for row in result]
+        return collectionGraph
```

Comment:
one thing you should ensure is that data is returned for all dates in the 90 days. if there is nothing in the sql result for that date it should be filled with 0